### PR TITLE
Ensure build action is set

### PR DESCRIPTION
**Pull request by MDrakos**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-2129" title="DX-2129" target="_blank">BugDX-2129</a>  runtime-debug: build action is not sent
  </td></table>
  <br />

### CODE REVIEW

File: pkg/platform/model/buildplanner.go [hunk 1]
```diff
@@ -53,7 +52,7 @@ type BuildResult struct {
 	CommitID            strfmt.UUID
 	Build               *bpModel.Build
 	BuildStatusResponse *headchef_models.V1BuildStatusResponse
-	BuildStatus         headchef.BuildStatusEnum
+	BuildStatus         string
 	BuildReady          bool
```
Comment (mitchell-as) [inline pkg/platform/model/buildplanner.go:56]: I wonder if `BuildReady` is necessary now that we can just test that `BuildStatus == bpModel.Complete`.

Comment (mitchell-as) [inline pkg/platform/model/buildplanner.go:56]: I wonder if `BuildReady` is necessary now that we can just test that `BuildStatus == bpModel.Complete`.

**Review by mitchell-as — APPROVED** [review]
Looks good to me. I did add a comment that may or may not be addressed.

File: pkg/platform/model/buildplanner.go (hunk 1 above)
Comment (MDrakos) [reply pkg/platform/model/buildplanner.go:56]: Yes, it likely can be replaced however I will leave that to be part of the upcoming runtime refactor.

**Pull request merged**